### ISSUE TITLE
Open edits shortcuts

### ISSUE THREAD

**Issue by uniquereplica** [jesus2099/konami-command#7]
Hi, great script, these changes are making a real difference to how easy it is to navigate and edit musicbrainz.

I was wondering if I could make a feature request, I hope it's not an issue.

On the edit page there is a menu as so:

    Show relate/merge tools
    Show Edit rec. buttons

I was wondering if it would be possible to add another menu item as follows:

    Show Open edit buttons

Unfortunately being new to this I am making a lot of mistakes, and cancelling relationships that I later decide are wrong is a real pain right now. It would be great if it the button showed up next to relationships as well as the recordings too.

As you can appreciate it saves clicking through on a relationship/track, waiting on the load, finding the open edits button, and clicking again. It may even help people avoid RSI/carpel tunnel syndrome! :-)

**Comment by jesus2099** [comment]
Hi @uniquereplica,

I come back here since it’s a very old issue ticket that you had kindly opened back then.
It’s quite unfair that I had kind of hijacked it to another broader feature that eventually is not coming quickly to concretion.

I have now created another ticket for my other idea (#222) and so let’s get back to the point with your issue.
I think you were simply asking for a new checkbox in INLINE STUFF for the release page, that I will be able to add more quickly.
And later, if #222 comes to concretion, I could remove that _temporary_ checkbox feature.
Sorry again and thanks for your interest, I hope you will still enjoy it when I fix it, quite soon enough. I hope you’re still here, using my script.

**Comment by jesus2099** [comment]
It’s there, @uniquereplica, at last! :)
